prijateli_tree/app/locales/en.yml

```diff
@@ -0,0 +1,112 @@
+
+  quiz_peers:
+    what_will_you_observe: "What will you observe to help you learn which bag the computer chose?"
+    one_ball_few_peers: "One ball and a few peers' guesses"
+    two_balls: "Two balls"
+    one_ball_all_peers: "One ball and all peers' guesses"
+
+  intro_practice:
+    now_you_will_play: "Now you will play 2 practice games to get used to the rules."
+    since_these_rules: "Since these are practice rounds, we will not reward anyone for their performance."
+
+  practice_game_video:
+    practice_game: "Practice Game"
+    explanation: "Explanation"
+    score: "Score"
+    feel_free: "Feel free to watch the video again to remind yourself of the rules."
+
+  practice_game_first_round:
+    round: "Round"
+    the_computer_chose: "The computer chose a bag and split the balls randomly to each player."
+    your_ball_is_red: "Your ball is RED"
+    your_ball_is_blue: "Your ball is BLUE"
+    guess_color: "Guess the color of the bag:"
+    red: "RED"
+    blue: "BLUE"
+
+  practice_game_other_rounds:
+    see_what_others_guessed: "See what some other players guessed in the last round:"
+    code_guessed_red: "Code guessed RED."
+    code_guessed_blue: "Code guessed BLUE."
+    name_guessed_red: "Name guessed RED."
+    name_guessed_blue: "Name guessed BLUE."
+    your_guess_was_red: "Your guess in the last round was RED."
+    your_guess_was_blue: "Your guess in the last round was BLUE."
+
+  practice_game_results:
+    results: "Results"
+    you_won: "You won!!!"
+    better_luck: "Better luck next time!"
+    the_computer_chose_red: "The computer chose the RED bag."
+    the_computer_chose_blue: "The computer chose the BLUE bag."
+
+
+  real_game_intro:
+    now_we_practiced: "Now that we practiced, we will play the real games."
+    get_ready: "Get ready to win and earn money!!!"
+    if_you_win: "If you win in this game, {X} points will be added to your score and final reward."
+    game: "Game"
+    added_points: "{X} points will be added to your final reward."
+
+
+  different_game_intro:
+    different_version: "Now, we will play a different version of the game."
+    instead_of: "Instead of assigning you the peers you observe randomly, we will let you choose the players you observe."
+
+  different_game_choices:
+    now_you_can_observe: "Now you can choose the players you will observe in the following rounds in this game."
+    please_choose_two_players: "Please choose two players"
+    please_choose_three_players: "Please choose three players"
+    player_one: "Player 1:"
+    player_two: "Player 2:"
+    player_three: "Player 3:"
+
+  survey:
+    before_we_are_done: "Before we are done, we want to ask you a few short questions about how you played today."
+    survey_page: "Survey Page"
+
+  final_score:
+    congratulations: "Congratulations!"
+    you_guessed_correctly: "You guessed correctly in {X} games, and earned {x} points, which means you made extra"
+    denars: "{X} denars!"
```

Comment:
Can these be used as f-strings? If so, it looks like these are good, but they'll otherwise not work.